reg/drone/README.md

```diff
@@ -0,0 +1,56 @@
+# Drone
+
+This namespace contains the application-specific regulated DSDL namespace for aerial vehicles.
+If you have any questions, feel free to bring them to the [UAVCAN Forum](https://forum.uavcan.org/c/sig/drone-sig/17).
+
```

Comment:
Can you please also reference the DS-015 document and point to it?

Reply:
Added a link to the Pixhawk-Standards repository, where I expect DS-015 will be published when ready. There is no point linking the draft document because it is, as of yet, out-of-sync with this work, and the internal one should stay internal. Let me know if I misinterpreted your intent here.